packages/teams-js/test/private/files.spec.ts

```diff
@@ -442,7 +442,7 @@ describe('files', () => {
     it('should trigger callback correctly', async () => {
       await utils.initializeWithContext('content');
 
-      const callback = jest.fn((err) => {
```

Comment:
These changes look like the issue Marcin pointed out about pulling from `main` and re-running `yarn` haven't been resolved yet.